### ISSUE TITLE
Monitoring jfrog artifactroy and integrating with grafana with influx db as data source.

### ISSUE THREAD

**Issue by simplyviks** [javamelody/javamelody#762]
How do we hook java melody to jfrog artifactory and send data to infux Db?

**Comment by evernat** [comment]
I suggest the following:
- open `webapps/artifactory.war`
- and add the [javamelody-core jar file](https://github.com/javamelody/javamelody/releases) and the [jrobin-1.5.9.jar file](https://repo1.maven.org/maven2/org/jrobin/jrobin/1.5.9/jrobin-1.5.9.jar) in WEB-INF/lib
- and in WEB-INF/web.xml, because there metadata-complete, add the following after `metadata-complete="true">`:
```xml
<filter>
	<filter-name>javamelody</filter-name>
	<filter-class>net.bull.javamelody.MonitoringFilter</filter-class>
	<async-supported>true</async-supported>
</filter>
<filter-mapping>
	<filter-name>javamelody</filter-name>
	<url-pattern>/*</url-pattern>
	<dispatcher>REQUEST</dispatcher>
	<dispatcher>ASYNC</dispatcher>
</filter-mapping>
<listener>
	<listener-class>net.bull.javamelody.SessionListener</listener-class>
</listener>
```

For influx db, see https://github.com/javamelody/javamelody/wiki/UserGuideAdvanced#sending-metrics-to-influxdb
Basically, you can add `javamelody.influxdb-url=http://11.22.33.44:8086/write?db=mydb&rp=myretentionpolicy` in `etc/artifactory.system.properties`
and it is also needed to [add log4j jar file](https://repo1.maven.org/maven2/log4j/log4j/1.2.17/log4j-1.2.17.jar) in WEB-INF/lib

**Comment by MohamedNazeer** [comment]
We have added the javamelody.influxdb-url parameter in etc/artifactory.system.properties but still javamelody not sending the metrics to influxdb.
